pkg/sanitize/sanitize.go

```diff
@@ -12,6 +12,30 @@ func String(str string) string {
 	return str
 }
 
+func SanitizeURLEncoding(str string) string {
```

Comment:
```suggestion
func URLEncoding(str string) string {
```
I would avoid adding the package name to the function